src/main.h

```diff
@@ -1303,6 +1303,7 @@ class CBlockIndex
     // Gridcoin (17.6.2017 Brod) Add dpor related back-pointers
     CBlockIndex* ppCpid;  // previous block by same cpid
     CBlockIndex* ppSuper; // previous superblock
+    CSuperblock* pDesSuper; // deserialized superblock data

```

Comment:
Is there any way we can keep this in CBlock and load it on demand? Trying my darndest to reduce the footprint of CBlockIndex :D